v_4_9_1/files/conf/k8s-addons

```diff
@@ -1,21 +1,20 @@
 #!/bin/bash
 
 export KUBECONFIG=/etc/kubernetes/kubeconfig/addons.yaml
-# kubectl 1.12.2
-KUBECTL={{ .RegistryDomain }}/giantswarm/docker-kubectl:f5cae44c480bd797dc770dd5f62d40b74063c0d7
+export KUBECTL="/usr/bin/docker run -e KUBECONFIG=${KUBECONFIG} --net=host --rm -v /srv:/srv -v /etc/kubernetes:/etc/kubernetes {{ .Images.Kubectl }}"
```

Comment:
This refactoring wasn't totally necessary, but it makes the file much more readable and therefore easier to debug/edit.